presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -480,6 +484,22 @@ public Boolean visitParameter(Parameter node, Void context)
             return process(parameters.get(node.getPosition()), context);
         }
 
+        public Boolean visitGroupingOperation(GroupingOperation node, Void context)
+        {
+            Collection<Expression> argumentsNotInGroupBy = node.getGroupingColumns().stream()
```

Comment:
Actually in the error message below I print out the arguments to `grouping()` that aren't part of the `group by` so I can't use `count()`. However I did switch `Collection` to a simple `List`.